chunker/chunk_documents_excel.py

```diff
@@ -0,0 +1,304 @@
+import logging
+import time
+import pandas as pd
+import os
+import re
+import glob
+import openpyxl
+from utils.file_utils import get_file_extension
+from utils.file_utils import get_filename
+
+
+
+FILE_EXTENSION_DICT = [
+    "xlsx",
+    "csv"
+    
+]
+
+##########################################################################################
+# UTILITY FUNCTIONS
+##########################################################################################
+
+def cleanup_title(title):
+
+    title = title.replace(" ", "%20")
+    
+    return title
+
```

Comment:
this method quote helps handle more than just space. Some times unusually characters may appear in the title as well.

```python 
from urllib.parse import quote

def cleanup_title(title):
    return quote(title)
```